### PR TITLE
Removed an outdated reference to a guide that no longer exists.

### PR DESCRIPTION
**Pull request by mig281**
The _Amazon SQS Getting Started Guide_ has been retired. The most current information is in the _Amazon SQS Developer Guide_: http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-tutorials.html

### CODE REVIEW

File: src/samples/AmazonSimpleQueueService/README.md [hunk 1]
```diff
@@ -28,4 +28,4 @@ The basic steps for running the Amazon SQS sample are:
 
 **NOTE:** The sample also includes an Ant build.xml file to run the sample.
 
-See the [Amazon SQS Getting Started Guide](http://docs.amazonwebservices.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/) for step-by-step instructions on running this sample.
+See the [tutorials](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-tutorials.html) in the _Amazon SQS Developer Guide_ for step-by-step description of this sample.
```
Comment (varunnvs92) [inline src/samples/AmazonSimpleQueueService/README.md]: The tutorials link is not a step-by-step description of this sample. May be we should change the link to "See the Amazon SQS Tutorials for more samples."

Comment (mig281) [inline src/samples/AmazonSimpleQueueService/README.md]: I don't know if I fully agree. Take, for example, [Send Message](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-send-message.html#send-message-java): the Java section does explain which section of the code does the sending, what exactly happens, and what values are returned. Moreover, the docs provide a way to do this with both standard and FIFO queues, which involves different code and which might be useful/interesting to someone who comes to that page.

Would you be willing to compromises on "for detailed descriptions of operating Amazon SQS using Java"?

Comment (varunnvs92) [inline src/samples/AmazonSimpleQueueService/README.md]: I don't like the word "operating". How about "for detailed description on how to work with Amazon SQS using Java."

**Comment by mig281** [comment]
@varunnvs92 Could you please merge the change?

**Comment by EronHennessey** [comment]
:+1:

**Pull request merged**